hazelcast/src/main/java/com/hazelcast/kubernetes/KubernetesClient.java

```diff
@@ -521,15 +521,19 @@ private static List<String> privateAddresses(List<Endpoint> endpoints) {
     }
 
     private static String extractLoadBalancerAddress(JsonObject serviceResponse) {
-        JsonObject ingress = serviceResponse
-                .get("status").asObject()
-                .get("loadBalancer").asObject()
-                .get("ingress").asArray().get(0).asObject();
-        JsonValue address = ingress.get("ip");
-        if (address == null) {
-            address = ingress.get("hostname");
-        }
-        return address.asString();
+        try {
+            JsonObject ingress = serviceResponse
+                    .get("status").asObject()
+                    .get("loadBalancer").asObject()
+                    .get("ingress").asArray().get(0).asObject();
+            JsonValue address = ingress.get("ip");
+            if (address == null) {
+                address = ingress.get("hostname");
+            }
+            return address.asString();
+        } catch (Exception e) {
```

Comment:
Do we want to catch all the exceptions, or would it be better to check for the `null` value explicitly?

Reply:
It is an option but I think it is better to wrap all exceptions with `KubernetesClientException`